src/main/resources/templates/change-creds.html

```diff
@@ -39,12 +39,12 @@ <h4></h4>
                         <h4 th:text="#{changeCreds.changeUserAndPassword}">Change Username and password</h4>
                         <form action="api/v1/user/change-username-and-password" method="post">
                             <div class="mb-3">
-                                <label for="newUsername" th:text="#{changeCreds.newUsername}">New Username</label>
-                                <input type="text" class="form-control" name="newUsername" id="newUsername" th:placeholder="${username}">
+                                <label for="username" th:text="#{changeCreds.username}">New Username</label>
```

Comment:
Need to update .newUsername to .username in translation files.

Reply:
Was this an intended change?

Reply:
It was not 
good catch 

Lazy copy pasting